### PR TITLE
feat: Add gcode converter

### PR DESCRIPTION
**Pull request by phbender**
I have

- modified the original CLI (inkdrop) to create JSON files as well (breaking change: parameter `--output` is now `--svg`)
- added a new CLI (gcode-converter) which reads this JSON, a calibration, and emits gcode which can be sent to the plotter

Reason is that the conversion to points takes long, the other is relatively quick, and to adjust the drawing etc. one does not want to run the image conversion again. Instead of adding a complex caching or the like, two steps may be simpler.

### CODE REVIEW

**Comment by phbender** [comment]
I have

- modified the original CLI (`inkdrop`) to create JSON files as well
- added a new CLI (`gcode-converter`) which reads this JSON, a calibration, and emits gcode which can be sent to the plotter

Reason is that the conversion to points takes long, the other is relatively quick, and to adjust the drawing etc. one does not want to run the image conversion again. Instead of adding a complex caching or the like, two steps may be simpler.

File: README.md [hunk 1]
```diff
@@ -57,6 +57,45 @@ gives us the following output:
...
+looks like this:
+
+```json
+{
+  "base_width": 300.0,
+  "base_height": 560.0,
+  "drawing_width": 300.0,
+  "drawing_height": 450.0
+}
+````
```
Comment (matze) [inline README.md:79]: Looks like the extra backtick breaks formatting.

**Comment by matze** [comment]
Looks good to me. Will merge when this `README.md` issue is solved.

**Comment by matze** [comment]
Totally forgot about this, will fix the formatting myself.

**Pull request merged**